tests/Container/ContainerTest.php

```diff
@@ -66,6 +66,19 @@ public function testBindIfDoesRegisterIfServiceNotRegisteredYet()
         $this->assertSame('Dayle', $container->make('name'));
     }
 
+    public function testBindWhenIfConditionIsTrue()
+    {
+        $container = new Container;
+        $container->bind('surname', function () {
```

Comment:
Shouldn't this be `name` instead of `surname`?

Reply:
No, under that is important

Reply:
Why is it there? I thought your were trying to show how a binding gets overridden.

Reply:
Fix it.